otx/cli/tools/train.py

```diff
@@ -128,6 +138,18 @@ def parse_args():
         type=float,
         help="Expected ratio of total time to run HPO to time taken for full fine-tuning.",
     )
+    parser.add_argument(
```

Comment:
Is there no default parameter for gpus?

Reply:
yes. it will be set only if user try to run multi gpu training.

Reply:
Maybe I missed something, is there any argument checking use of multi gpu training? What happens when user don't give gpus argument(he wants single gpu training)

Reply:
then gpus argument is set as None, and single gpu training will start.